tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -150,59 +159,101 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
-                        jqUnit.assertNode("The contents of the tooltip should be set", tooltipContents[0], $("b", tooltip));
+                        jqUnit.assertNode("When the page size is " + pager.model.pageSize + ", The contents of the tooltip should be set", expectedTooltipContents[index], $("b", tooltip));
                     });
                 }
             });
         };
 
-        var tooltipTest = function (location) {
-            return function (idx, linkEl) {
-                var linkId = linkEl.id;
-                var link = $(linkEl);
+        var tooltipTest = function (location, index, expectedTooltipContents) {
+            var link =  "#rendered-ioc .flc-pager-" + location + " .flc-pager-links .flc-pager-pageLink a:eq(" + index + ")";
+            var linkNumber = index + 1;
+            var linkId = "page-link:link" + linkNumber + (location === "bottom" ? "-1" : "");
+            sequence.push({
+                element: link,
+                jQueryTrigger: "focus"
+            });
+            assertVisibleTips("When the page size is " + pager.model.pageSize + ", The tooltip for page link " + linkNumber + ", in the " + location + " page bar is visible", [linkId], expectedTooltipContents, index);
+            sequence.push({
+                element: link,
+                jQueryTrigger: "blur"
+            });
+            assertVisibleTips("There shouldn't be any tooltips visible when none of the pageLinks are focused", [], expectedTooltipContents, index);
+        };
+
+        /* 
+         * Change the page size if it has provided and
+         * verify whether the displaying links and their tooltips are synced are with page size
+         */
+        var verifyToolTips = function (pageSize) {
+            if (pageSize) {
                 sequence.push({
-                    element: link,
-                    jQueryTrigger: "focus"
+                    func: function () {
+                        $("#rendered-ioc .flc-pager-page-size").val(pageSize);
+                    }
                 });
-
-                if (link.hasClass(pager.pagerBar.options.styles.currentPage)) { // assumption that this will not change between now and then!
-                    assertVisibleTips("There shouldn't be any tooltips visible when the currentPage is focused", []);
-                } else {
-                    assertVisibleTips("Only the tooltip for page link " + (idx + 1) + ", in the " + location + " page bar is visible", [linkId]);
-                }
                 sequence.push({
-                    element: link,
-                    jQueryTrigger: "blur"
+                    element: "#rendered-ioc .flc-pager-page-size",
+                    jQueryTrigger: "change"
                 });
-                assertVisibleTips("There shouldn't be any tooltips visible when none of the pageLinks are focused", []);
-            };
+            } else {
+                pageSize = pager.model.pageSize;
```

Comment:
@cindyli @jobara 

This is the use case where the test is executed without changing the page size and considering the current page size. Which mean if the pageSize parameter has not passed.